services/initdaily.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python3 $HOME/cassandra-GLAM-tools/services/daily.py  -e development
```

Comment:
in prod the env var should be "production", "development" is only for local development

Reply:
I changed it, and I also changed it in the README